guids-generator.html

```diff
@@ -114,11 +114,11 @@
             return MOD_10_check_digit(value) ? value : {"error" : "Health card number " + value + " is invalid for the province of ON. Please check the number and try again."};
         },
         QC: function (value, dob) {
-            console.log(dob);
+            let disableValidateChecksum = false;
```

Comment:
Why do we introduce it?

Reply:
For convenience in case we want to disable it easily.